packages/amplify-migration-tests/src/migration-helpers-v10/init.ts

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
I created this folder so that we can place v10 e2e tools here.. such as for init, and other custom e2e flows